src/libraries/System.Runtime.Caching/tests/System.Runtime.Caching/MemoryCacheTest.cs

```diff
@@ -969,7 +969,8 @@ public void ChangeMonitors()
         }
 
         // Due to internal implementation details Trim has very few easily verifiable scenarios
-        [Fact]
+        // ActiveIssue: https://github.com/dotnet/runtime/issues/36488
+        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsNotArm64Process))]
```

Comment:
I think I understand why we failed here - can we fix the test instead of disabling it?

Reply:
So your suggestion is to apply a trim of 76% instead?

Reply:
Yes. Having 4 items out of 30 with the same hash code is already extremely rare but it happened. So with 76% trim this would have to be 5 items having the same hash... 
Let's give it a try - maybe my understanding is completely wrong.

Reply:
It doesn’t seem to be failing every time though, seems more flaky. Should we be more flexible and maybe accept both as valid? Either the result of 75% and 76%?

Reply:
Yes, the test is flaky... My guess is because the hash code is non stable between process restarts.
Does it fail at all with 76% trim? It would take 5 strings to have the same remainder of their hash code which should be more rare than 4...

Reply:
Will merge this as is in order to unblock CI and will submit various jobs to the test queue with your suggestion to see if that fixes the issue. Will update the issue later today.